Eevee.Sleep.Bot/Controllers/Mongo/MongoConst.cs

```diff
@@ -20,4 +20,9 @@ public static class MongoConst {
 
     public static readonly IMongoCollection<ActivationPresetModel> AuthActivationPresetCollection =
         AuthDatabase.GetCollection<ActivationPresetModel>("activationPreset");
+
+    private static readonly IMongoDatabase RoleDatabase = Client.GetDatabase("role");
```

Comment:
Change database name to `discord`

Reply:
Fixed!
[9a44466](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/9a44466615d65b9af287798a6c556c2aad368643)